templates/flask/files/.platform.template.yaml

```diff
@@ -36,6 +36,7 @@ info:
   notes:
     - heading: "Apps & Services"
       content: "Python 2.7<br/>MySQL 10.2<br/>Redis 3.2"
+    - heading: "Requirements"
```

Comment:
Hm, we just removed this line from the template files.  Merge issue or old data, maybe?